git-extra/astextplain

```diff
@@ -8,6 +8,9 @@ fi
 
 # XXX output encoding (UTF-8) hardcoded
 case "$1" in
+	*.ods | *.ODS | *odf |*.ODF | *odt | *.ODT)
+		odf2txt "$1" || cat "$1"
```

Comment:
I tried to explain the difference here: https://github.com/git-for-windows/git/wiki/The-difference-between-MINGW-and-MSYS2 (sorry for the link, but I found that it is not an easy thing to explain, so I spent something like a dozen hours to try to explain it well, once and for all).